app/options/defaults.ts

```diff
@@ -5,6 +5,7 @@ export default {
   localStorage: true,
   askConfirmation: false,
   closeTab: false,
+  closeSameDomainTabs: true,
```

Comment:
Since it's a new option, the default should be `false` to prevent people losing data.

Please also add the option to https://github.com/blaise-io/forget-about-this-site/blob/master/app/options/options.ejs so users can modify the default.

This will introduce a new `data-message` that we would have to translate in https://github.com/blaise-io/forget-about-this-site/tree/master/app/_locales (using Deepl.com and Google Translate).